tests/integration/parity/common.py

```diff
@@ -168,6 +168,22 @@ def test_eth_getLogs_without_logs(self, web3, block_with_txn_with_log):
         result = web3.eth.getLogs(filter_params)
         assert len(result) == 0
 
+    def test_eth_signTypedData(self,
+                               web3,
+                               unlocked_account_dual_type):
+        pytest.xfail('eth_signTypedData JSON RPC call has not been released in parity')
+        super().test_eth_signTypedData(
+            web3, unlocked_account_dual_type
+        )
+
+    def test_invalid_eth_signTypedData(self,
+                                       web3,
+                                       unlocked_account_dual_type):
+        pytest.xfail('eth_signTypedData JSON RPC call has not been released in parity')
```

Comment:
Is there a reason these are not using the decorator method of marking xfails?  That approach is superior because it still has pytest run the test and reports un-expected test success as a failure to ensure that all of the `xfail` tests actually fail (rather than this approach which will cause the test to unconditionally fail).

Reply:
I changed it to the decorator for these tests, but the rest of the file has the `pytest.xfail` pattern. I added an issue to switch them over throughout the codebase (#1341)